Week4/README.md

```diff
@@ -95,3 +95,32 @@ There are TWO ways callbacks can be set.
 </script>
 ```
 
+### Important Javascript Methods
+
+This website is a great resource: [javascripttutorial.net](https://www.javascripttutorial.net/javascript-dom/). Below are just a few of the highlights.
+
+The term "element" and "tag" are used interchangeably below
+
+#### Getting Elements
+
+- *getElementById("id")*<p>Gets an element by its id attribute. Remember ID is unique, so this will only get ONE element at a time 
+- *getElementsByTagName("tag")*<p>Gets all the elements with a given Tag Name, e.g. getElementsByTagName("h2")
+- *getElementsByQuerySelector("selector")*<p>Lets you use a css selector to get elements, e.g. getElementsByQuerySelector("h1 h2 h3") will get all the `<h1>`,`<h2>`, and `<h3>` tags
```

Comment:
I've never heard of `getElementsByQuerySelector`. There is `DOMElement.querySelector`, but it only selects the first element that matches the query selector. Also, the selector, `h1 h2 h3` means any `h3` that is a (possibly distant) child of an `h2` that is a (possibly distant) child of an `h1`, but I don't think query selectors work on nested header elements? If you want all `h1`s, `h2`s, and `h3`s, you should probably use `document.querySelectorAll('h1, h2, h3');`.